tests/system/src/test/java/test/robot/javafx/scene/PixelBufferDrawTest.java

```diff
@@ -89,7 +89,7 @@
     private static Robot robot;
     private static CountDownLatch startupLatch;
 
-    private static final int DELAY = 100;
+    private static final int DELAY = 200;
```

Comment:
I ran it 100 times and got 1 failure on my slow Linux box. Can you increase to 500 msec?

Reply:
changed delay to 500